1.hello-world/README.md

```diff
@@ -135,7 +135,7 @@ You're up and running! Both Dapr and your app logs will appear here.
 ```
 > **Note**: the `--app-port` (the port the app runs on) is configurable. Our Node app happens to run on port 3000, but we could configure it to run on any other port. Also note that the Dapr `--port` parameter is optional, and if not supplied, a random available port is used.
 
-The `dapr run` command looks for a `components` directory which holds yaml definition files for components Dapr will be using at runtime. When running locally, if the directory is not found it is created with yaml files which provide default definitions for a local development environment (learn more about this flow [here](https://github.com/dapr/docs/blob/master/walkthroughs/darprun.md)). Review the `statestore.yaml` file in the `components` directory:
+The `dapr run` command looks for the default components directory under `~/.dapr/components` which holds yaml definition files for components Dapr will be using at runtime. When running locally, the yaml files which provide default definitions for a local development environment are placed in this default components directory (learn more about this flow [here](https://github.com/dapr/docs/blob/master/walkthroughs/darprun.md)). Review the `statestore.yaml` file in the `components` directory:
```

Comment:
For samples, please look at all the samples which run locally, update the commands int heir documentation to specify the components directory which is needed by that sample. One example is Binding sample which uses Kafka binding https://github.com/dapr/samples/tree/master/5.bindings

Reply:
fixed. The other samples do not seem to require a special components folder